03-use-cases/01-finance/retail-banking/main.ipynb

```diff
@@ -0,0 +1,1931 @@
+{
```

Comment:
<p>Please add explanations or business context here.</p><p> </p><p>After these, please add visualizations to help people understand what these measures look like. The number of cells being added won't be more onerous than the mental gymnastics of someone trying to just follow measure creation without any visuals.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/824/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>